src/OrchardCore.Modules/OrchardCore.Contents/Views/Tab.cshtml

```diff
@@ -7,6 +7,7 @@
     Model.Metadata.Alternates.Clear();
     Model.Metadata.Type = "CardGrouping";
     Model.Classes.Clear();
+    Model.Classes.Add("accordion");
```

Comment:
See the issue here is that this should be defined from the .cshtml file to make it agnostic from the current Bootstrap styles. So, this is why mainly I did not hardcode it in the Shape. We need to find a better place to add these styles then. I mean, maybe we are missing a wrapper shape.

Reply:
I will think about this one and try to find a solution. Here the issue is that a CardGrouping needs to be displayed as an accordion "by design" because we are missing bits to define a DisplayType on the CardGrouping. It doesn't need to be defined from the Views if we define it from the Driver...

Reply:
![image](https://user-images.githubusercontent.com/3228637/109398515-6e79a880-790b-11eb-999f-cb02e6016afe.png)

Reply:
Something like this : 

![image](https://user-images.githubusercontent.com/3228637/109398536-7e918800-790b-11eb-8dfd-6d83717bba5c.png)